config/initializers/lograge.rb

```diff
@@ -2,4 +2,6 @@
 
 Rails.application.configure do
   config.lograge.enabled = Rails.env.production?
+  config.lograge.base_controller_class = ['ActionController::API', 'ActionController::Base']
+  config.lograge.ignore_actions = ['OkComputer::OkComputerController#show']
```

Comment:
Do we need to see each GET request Nagios sends to the OkComputer controller every minute?